### PR TITLE
misc changes from QA

### PR DESCRIPTION
**Pull request by acharb**
bunch of unrelated fixes so I added the QA ticket/figma convo for each change

### CODE REVIEW

File: extension/src/popup/components/sendPayment/SendAmount/SendType/index.tsx [hunk 1]
```diff
@@ -6,7 +6,6 @@ import { Button, Icon } from "@stellar/design-system";
 import { navigateTo } from "popup/helpers/navigate";
 import { ROUTES } from "popup/constants/routes";
 import { PopupWrapper } from "popup/basics/PopupWrapper";
-import { AutoSaveFields } from "popup/components/AutoSave";
```
Comment (acharb) [inline extension/src/popup/components/sendPayment/SendAmount/SendType/index.tsx:9]: removing auto save
https://stellarorg.atlassian.net/browse/WAL-126?filter=10043

File: extension/src/popup/components/sendPayment/SendAmount/index.tsx [hunk 2]
```diff
@@ -101,9 +106,21 @@ export const SendAmount = ({ previous }: { previous: ROUTES }) => {
     navigateTo(ROUTES.sendPaymentSettings);
   };
 
+  const validate = (values: { amount: string }) => {
```
Comment (acharb) [inline extension/src/popup/components/sendPayment/SendAmount/index.tsx:109]: not allowing continue if error with send amount

File: extension/src/popup/components/sendPayment/SendConfirm/TransactionDetails/index.tsx [hunk 3]
```diff
@@ -34,13 +34,7 @@ import { AccountAssets } from "popup/components/account/AccountAssets";
 
 import "./styles.scss";
 
-export const TransactionDetails = ({
```
Comment (acharb) [inline extension/src/popup/components/sendPayment/SendConfirm/TransactionDetails/index.tsx:37]: transaction details "back" should go to accountHistory
https://stellarorg.atlassian.net/browse/WAL-128?filter=10043

File: extension/src/popup/components/sendPayment/SendSettings/Slippage/index.tsx [hunk 4]
```diff
@@ -62,8 +63,11 @@ export const SendSettingsSlippage = () => {
             );
             navigateTo(ROUTES.sendPaymentSettings);
           }}
+          validationSchema={YupObject().shape({
```
Comment (acharb) [inline extension/src/popup/components/sendPayment/SendSettings/Slippage/index.tsx:66]: maxing slippage to 10%
https://www.figma.com/file/BR8lyECrgUHybq7XuQSEED?node-id=1693:4012#172895146

File: extension/src/popup/components/sendPayment/SendSettings/TransactionFee/index.tsx [hunk 5]
```diff
@@ -104,7 +104,7 @@ export const SendSettingsFee = () => {
                       setFieldValue("transactionFee", recommendedFee)
                     }
                   >
-                    Set default
+                    Set recommended
```
Comment (acharb) [inline extension/src/popup/components/sendPayment/SendSettings/TransactionFee/index.tsx:107]: https://www.figma.com/file/BR8lyECrgUHybq7XuQSEED?node-id=1622:5726#173517404

File: extension/src/popup/components/sendPayment/SendTo/index.tsx [hunk 6]
```diff
@@ -196,7 +196,12 @@ export const SendTo = ({ previous }: { previous: ROUTES }) => {
         title="Send To"
         customBackAction={() => navigateTo(previous)}
       />
-      <form>
+      <form
```
Comment (acharb) [inline extension/src/popup/components/sendPayment/SendTo/index.tsx:199]: fixing how pressing enter on the SendTo and SendAmount inputs submits the form by mistake
https://stellarorg.atlassian.net/browse/WAL-125?filter=10043

File: extension/src/popup/views/ViewPublicKey/index.tsx [hunk 7]
```diff
@@ -63,37 +64,44 @@ export const ViewPublicKey = () => {
...
-              ) : (
-                <>
-                  <div className="ViewPublicKey__account-name-display">
-                    {accountName}
-                  </div>
-                </>
-              )}
-              <div className="ViewPublicKey--account-name-div">
-                <EditNameButton />
+          <Formik
```
Comment (acharb) [inline extension/src/popup/views/ViewPublicKey/index.tsx:67]: max characters for account name
https://stellarorg.atlassian.net/browse/WAL-96?filter=10043

File: extension/src/popup/views/ViewPublicKey/styles.scss [hunk 8]
```diff
@@ -4,6 +4,7 @@
   align-items: center;
 
   &__header {
+    height: 2rem;
```
Comment (acharb) [inline extension/src/popup/views/ViewPublicKey/styles.scss:7]: fix account name moving when editing
https://stellarorg.atlassian.net/browse/WAL-95?filter=10043

File: extension/src/popup/views/ViewPublicKey/index.tsx (hunk 7 above)
Comment (acharb) [reply extension/src/popup/views/ViewPublicKey/index.tsx:67]: one of the subtasks for this is adding a disable horizontal scroll: https://stellarorg.atlassian.net/browse/WAL-97?filter=10043

my thoughts are that _ideally_ there shouldn't be a place that causes the screen to grow horizontally outside the popup. But if we miss a spot, then we'd want to allow scroll so important content doesn't get cut off.
(eg. if we missed this account name max, and didn't allow scroll then the edit name button would be hidden and they'd be left unable to edit their name.)

wdyt?

Comment (piyalbasu) [reply extension/src/popup/views/ViewPublicKey/index.tsx:69]: Yeah, agreed. In the case of a bug, I'd rather have the UI look a little ugly than cut off content. I'll comment on the ticket

**Review by piyalbasu — APPROVED** [review]

**Comment by acharb** [comment]
changed to allow 24 char account names:https://stellarorg.atlassian.net/browse/WAL-96
https://github.com/stellar/freighter/pull/418/commits/5a696a8e0b5ba2008a178cf873269a72552c3713

**Pull request merged**